components/admin_console/team_channel_settings/abstract_list.jsx

```diff
@@ -108,7 +108,7 @@ export default class AbstractList extends React.PureComponent {
         const firstPage = this.state.page === 0;
         return (
             <div className='groups-list groups-list-no-padding'>
-                {total > 0 && this.props.header}
```

Comment:
Need to show the header always (even when there are no results) so that the search input (inside the header) is available when there are no results from the search.